src/Calculator.cpp

```diff
@@ -8,4 +8,9 @@
 
 #include "Calculator.h"
 
+std::string Calculator::hello()
```

Comment:
Add this to separate private and public methods inside of the C++ only
//*************/
// Calculator private methods /
///